flatpak/io.github.softfever.OrcaSlicer.yml

```diff
@@ -221,6 +195,12 @@ modules:
         dest: external-packages/GMP
         sha256: eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c
 
+      # curl
```

Comment:
For CURL and OpenSSL, any reason why you're building those? The runtime versions should be new enough (curl 8.13.0 and openssl 3.3.3).

Reply:
I don't think there is a way to disable building these. No one backported your patches from Bambu Studio for disabling built-in deps